h/streamer/filter.py

```diff
@@ -52,9 +54,12 @@ def matching(cls, sockets, annotation):
 
             # Iterate over the filter_rows added by `set_filter()`
             for field, value in socket.filter_rows:
-                if value in values[field]:
-                    yield socket
-                    break
+                try:
+                    if value in values[field]:
+                        yield socket
+                        break
+                except KeyError:
+                    continue
```

Comment:
Don't barf on bad fields

Reply:
AIUI `filter_rows` _should_ only be set with `set_filter`, so any unexpected entries here would indicate an error in `set_filter`?

Reply:
Yes... this should be safe for now, but saves us from a code change in `set_filter` causing a seemingly unrelated explosion.